tests/integ/sagemaker/lineage/conftest.py

```diff
@@ -664,6 +664,9 @@ def static_endpoint_context(sagemaker_session, static_pipeline_execution_arn):
         "ContextSummaries"
     ]
     if len(contexts) != 1:
+        for ctx in context:
+            logging.info(f'Found context "{ctx["ContextArn"]}"')
```

Comment:
Can we change the both logger statements to `logging.error` with some more details to help debugging.

Reply:
sure